cmd/createtree/main.go

```diff
@@ -81,18 +81,41 @@ func createTree(ctx context.Context) (*trillian.Tree, error) {
 	defer conn.Close()
 
 	client := trillian.NewTrillianAdminClient(conn)
+	var tree *trillian.Tree
 	for {
-		tree, err := client.CreateTree(ctx, req)
-		if err == nil {
-			return tree, nil
+		glog.Info("CreateTree...")
```

Comment:
I think the pattern we want here looks a little more like:
```
for {
   select {
      case <- ctx.Done():
            return ctx.Err()
      case <- time.After(backoff.Time())
              // Retry
   }
}
```

Reply:
Happy to modify the pre-existing pattern to a friendlier backoff like you propose, but I'd rather do that in a follow up to keep this delta as small as possible.

Reply:
Sounds good